src/NServiceBus.Persistence.CosmosDB.TransactionalSession.AcceptanceTests/TransactionSessionDefaultServer.cs

```diff
@@ -0,0 +1,25 @@
+namespace NServiceBus.TransactionalSession.AcceptanceTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Support;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+
+    public class TransactionSessionDefaultServer : DefaultServer
+    {
+        public override Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor, EndpointCustomizationConfiguration endpointConfiguration,
+            Func<EndpointConfiguration, Task> configurationBuilderCustomization) =>
+            base.GetConfiguration(runDescriptor, endpointConfiguration, async configuration =>
+            {
+                // Explicitly enforcing the type to be scanned. Otherwise the scanner would not pick it up in the acceptance tests
+                endpointConfiguration.TypesToInclude.Add(typeof(TransactionalSession));
+                endpointConfiguration.TypesToInclude.Add(typeof(CosmosTransactionalSession));
+
```

Comment:
shouldn't this call `EnableTransactionalSession` somewhere (based on the template name)?

Reply:
It is here https://github.com/Particular/NServiceBus.Persistence.CosmosDB/pull/417/files#diff-fff055b14a94240fcbf10059438c02faa41223f7183a90d3b372bd7b83e7e887R20 or maybe I misunderstood your comment

Reply:
Ah now I understand. You are saying it should be placed here instead of the Configure class. It would be doable if we introduce the concrete ConfigureXYZPersistence() methods that return the persistence extensions. I'm happy to go with this approach. Let me know if you prefer that